contributors/design-proposals/cpu-manager.md

```diff
@@ -145,6 +145,10 @@ configuration. The three policies are **none**, **static** and **dynamic**.
 
 The active CPU manager policy is set through a new Kubelet
 configuration value `--cpu-manager-policy`. The default value is `none`.
+This policy will be reported in the Node resource as a label with key
+`alpha.kubernetes.io/cpu-policy`.  This allows users with with a policy
+preference to use a node selector to ensure the pod lands on a node with a
+particular cpu manager policy enabled.
```

Comment:
using labels for this could still work, I would just expect them to be applied to nodes, rather than having nodes self-report them